db/db_impl.cc

```diff
@@ -4687,10 +4669,6 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
     // we are a non-leader in a parallel group
     PERF_TIMER_GUARD(write_memtable_time);
 
-    if (log_used != nullptr) {
-      *log_used = w.log_used;
```

Comment:
This is done on STATE_COMPLETED (if it is a follwer) and end of WriteImpl (if it is a leader). Removing duplicate.